### DISCUSSION TITLE
Browser edition?

### DISCUSSION THREAD

**Discussion by FlyGues2889** [material-components/material-web#5721] [Q&A]
Is there a way to use all components in browser? Just like importing a few files.

**Comment by asyncliz** [comment]
Yes, you can use the components directly from a CDN. There's a full example in our [quick start guide](https://github.com/material-components/material-web/blob/main/docs/quick-start.md).

Example: https://jsbin.com/nimafiqahu/edit?html,output

You can import from `@material/web/all.js` or `@material/web/common.js` for all or most common components, or import them individually.
